tests/test_oci_features.py

```diff
@@ -0,0 +1,229 @@
+            "CAP_IPC_OWNER",
+            "CAP_SYS_MODULE",
+            "CAP_SYS_RAWIO",
+            "CAP_SYS_CHROOT",
+            "CAP_SYS_PTRACE",
+            "CAP_SYS_PACCT",
+            "CAP_SYS_ADMIN",
+            "CAP_SYS_BOOT",
+            "CAP_SYS_NICE",
+            "CAP_SYS_RESOURCE",
+            "CAP_SYS_TIME",
+            "CAP_SYS_TTY_CONFIG",
+            "CAP_MKNOD",
+            "CAP_LEASE",
+            "CAP_AUDIT_WRITE",
+            "CAP_AUDIT_CONTROL",
+            "CAP_SETFCAP",
+            "CAP_MAC_OVERRIDE",
+            "CAP_MAC_ADMIN",
+            "CAP_SYSLOG",
+            "CAP_WAKE_ALARM",
+            "CAP_BLOCK_SUSPEND",
+            "CAP_AUDIT_READ",
+            "CAP_PERFMON",
+            "CAP_BPF",
+            "CAP_CHECKPOINT_RESTORE"
+        ],
+        "cgroup": {
+            "v1": True,
+            "v2": True,
+            "systemd": True,
+            "systemdUser": True
+        },
+        "seccomp": {
+            "enabled": True,
+            "actions": [
+                "SCMP_ACT_ALLOW",
+                "SCMP_ACT_ERRNO",
+                "SCMP_ACT_KILL",
+                "SCMP_ACT_KILL_PROCESS",
+                "SCMP_ACT_KILL_THREAD",
+                "SCMP_ACT_LOG",
+                "SCMP_ACT_NOTIFY",
+                "SCMP_ACT_TRACE",
+                "SCMP_ACT_TRAP"
+            ],
+            "operators": [
+                "SCMP_CMP_NE",
+                "SCMP_CMP_LT",
+                "SCMP_CMP_LE",
+                "SCMP_CMP_EQ",
+                "SCMP_CMP_GE",
+                "SCMP_CMP_GT",
+                "SCMP_CMP_MASKED_EQ"
+            ],
+            "archs": [
+                "SCMP_ARCH_AARCH64",
+                "SCMP_ARCH_ARM",
+                "SCMP_ARCH_MIPS",
+                "SCMP_ARCH_MIPS64",
+                "SCMP_ARCH_MIPS64N32",
+                "SCMP_ARCH_MIPSEL",
+                "SCMP_ARCH_MIPSEL64",
+                "SCMP_ARCH_MIPSEL64N32",
+                "SCMP_ARCH_PPC",
+                "SCMP_ARCH_PPC64",
+                "SCMP_ARCH_PPC64LE",
+                "SCMP_ARCH_RISCV64",
+                "SCMP_ARCH_S390",
+                "SCMP_ARCH_S390X",
+                "SCMP_ARCH_X32",
+                "SCMP_ARCH_X86",
+                "SCMP_ARCH_X86_64"
+            ]
+        },
+        "apparmor": {
+            "enabled": True
+        },
+        "selinux": {
+            "enabled": True
+        }
+    },
+    "annotations": {
+        "io.github.seccomp.libseccomp.version": "2.5.3",
+        "org.opencontainers.runc.checkpoint.enabled": True,
+        "run.oci.crun.checkpoint.enabled": True,
+        "io.github.containers.crun.commit": "0da15fb20f5ef55166e679862a0a7cca5fe6021a",
+        "io.github.containers.crun.version": "1.8.5.0.0.0.30-7da99"
+    }
+}
+
+def test_crun_features():
+    try:
+        output = run_crun_command(["features"])
+        features = json.loads(output)
+        if features == expected_features:
```

Comment:
I think this is too restrictive, we could just verify few attributes instead of expecting the output to be exactly the same as the static version defined above